### PR TITLE
Updates to MTR

### PR DESCRIPTION
**Pull request by OR13**

### CODE REVIEW

File: docs/openapi/components/schemas/common/ChemicalComposition.yml [hunk 1]
```diff
@@ -0,0 +1,153 @@
+$linkedData:
+  term: ChemicalComposition
+  '@id': https://w3id.org/traceability#ChemicalComposition
+title: Chemical Composition
+description: Measured chemical composition; all values in W%.
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/ChemicalComposition.yml:5]: Im guessing W% is a typo?

**Review by mkhraisha — COMMENTED** [review]
We have: https://github.com/w3c-ccg/traceability-vocab/blob/main/docs/openapi/components/schemas/common/ChemicalProperty.yml 
for chemical Properties, and now we're adding chemical composition, which seems to be steel specific. 
We should keep in mind that other items may need a chemical composition and that it would require them to add their own terms to this. 
Is there a way we can use Chemical Property to make it more extensible?

File: docs/openapi/components/schemas/common/ChemicalComposition.yml (hunk 1 above)
Comment (TallTed) [reply docs/openapi/components/schemas/common/ChemicalComposition.yml:5]: I think it stands for "[Weight percentage](https://www.google.com/search?q=%22W%25%22+weight)"... which would probably be better written out, here.
```suggestion
description: Measured chemical composition; all values in weight percentage (W%). 
```

**Comment by nissimsan** [comment]
Yes, @TallTed , weight percentage - thank you for clarifying!

**Review by BenjaminMoe — APPROVED** [review]

**Review by nissimsan — APPROVED** [review]

**Review by BenjaminMoe — CHANGES_REQUESTED** [review]
Should be inlined.

**Review by mkhraisha — APPROVED** [review]

**Review by BenjaminMoe — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**